visualization/app/codeCharta/codeCharta.component.html

```diff
@@ -19,15 +19,13 @@
     <sidenav-toggle-component></sidenav-toggle-component>
 </div>
 
-<code-map-component ng-hide="$ctrl._viewModel.isLoadingFile"></code-map-component>
```

Comment:
We should try to keep the map hidden again if a file is loading

Reply:
I implemented the `LoadingGifSubscriber` inside the `codeMap.component`